### ISSUE TITLE
Windows: uninstall should delete config

### ISSUE THREAD

**Issue by Quackdoc** [nushell/nushell#8053]
### Describe the bug

none of the internal commands work. I have tried uninstalling and re-installing nushell. 

### How to reproduce

1.  run a command like ls 

### Expected behavior

list directories 

### Screenshots

![image](https://user-images.githubusercontent.com/74831516/218305817-679bf005-4f71-4572-b4f6-0b6874f29465.png)

### Configuration

no output

**Comment by sholderbach** [comment]
Ah I suspect that this comes down to changed handling of the hook `display_output: $nothing`. I think `config reset --nu` could reset your `config.nu` to get there.

**Comment by Quackdoc** [comment]
that worked, thanks a bunch, I would close this, but I think a re-install of nu should probably do this automatically? or rather a full uninstall should clear any of the settings that could cause this.

**Comment by rgwood** [comment]
Yeah, it would probably make sense for an uninstall of the MSI to delete `config.nu`.

**Comment by hustcer** [comment]
I'm not sure if deleting configuration files when uninstalling an application is the right thing to do. Doing so might mistakenly remove the user's personalized settings, leading to more serious consequences. It's probably best to let users decide for themselves whether to delete the configuration files by running `config reset --nu`, instead of deleting them automatically during uninstallation.

**Comment by Kissaki** [comment]
I would expect an uninstall to keep the config and env files. That allows uninstall + install while preserving user settings.

The uninstaller could offer to remove the config files, not active by default.

The uninstaller could also offer running config reset as an alternative to uninstalling, which would make this option more visible, and visible as an alternative debugging/reset with issues at hand.

**Comment by hustcer** [comment]
In order to keep the uninstaller simple and easier to maintain, as well as to keep the uninstall logic consistent with the Linux installation package, I suggest leaving this issue as it is and not making any changes.
